### ISSUE TITLE
Using `styled-components/native` with last RN throws `Attempted import error: 'react-native' does not contain a default export (imported as 'reactNative')`

### ISSUE THREAD

**Issue by morissonmaciel** [styled-components/styled-components#2674]
### System:
 - OS: macOS 10.14.5
 - CPU: (4) x64 Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz
 - Memory: 22.78 MB / 8.00 GB
 - Shell: 3.2.57 - /bin/bash
### Binaries:
 - Node: 12.5.0 - /usr/local/bin/node
 - npm: 6.10.0 - /usr/local/bin/npm
### npmPackages:
 - styled-components: ^4.3.2 => 4.3.2

## Steps to reproduce
1. Create new project with `expo init my-project`
2. Access `my-project` folder and run `npm i styled-components`
3. Create a simple component with `import styled from "styled-components/native"` instruction
4. Import/use this component in `App.js`
5. Execute `expo start`

## Expected Behavior
Component rendering normally

## Actual Behavior
The following warning is logged:
```
Compiled with warnings.
Attempted import error: 'react-native' does not contain a default export (imported as 'reactNative').
```

**Comment by morissonmaciel** [comment]
I managed to run my app without issues just correcting `import` syntax in `node_modules/styled-components/native/dist/styled-components.native.esm.js`

from:
```js
import reactNative, { StyleSheet } from "react-native"
```
to:
```js
import * as reactNative from "react-native"; 
import { StyleSheet } from 'react-native';
```

**Comment by RikuVan** [comment]
Have same issue.  @morissonmaciel are you planning on doing a PR?

**Comment by RikuVan** [comment]
I guess since I am using react-native-web this issue is being handled elsewhere, e.g. [#2624](https://github.com/styled-components/styled-components/issues/).

**Comment by acro5piano** [comment]
I had the same problem with @RikuVan and upgrading styled-components to `v5.0.0-beta.8` solved it.

**Comment by hype08** [comment]
can confirm the suggestion by @acro5piano worked for me! 🙌  

i looked at #2624 and others suggested above, but upgrading to styled-components was easiest

"expo": "^35.0.0",
 "react": "16.11.0",
 "react-dom": "16.11.0",
 "react-native": "https://github.com/expo/react-native/archive/sdk-35.0.0.tar.gz",
 "react-native-web": "^0.11.7",

styled-components from 4.4 to v5.0.0-beta.8

**Comment by probablyup** [comment]
Fixed by 4cf2c5af04f1ebe2941ee3336d25b6737999ba85 and will come out in 4.4.1 soon
